cmd/prettyjson.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/cloudingcity/gool/pkg/prettyjson"
+	"github.com/spf13/cobra"
+)
+
+var prettyJSONCmd = &cobra.Command{
+	Use:                   "pretty-json [content]",
```

Comment:
```suggestion
	Use:                   "format-json [text]",
```
And put in format.go

Maybe we will have format-xml format-sql cmd.

Reply:
Great.

I have been done for this suggestion